deploy/job-manager/job-manager-deployment.yaml.ctmpl

```diff
@@ -6,6 +6,7 @@ metadata:
     app: job-manager
 spec:
   replicas: {{ env "REPLICAS" }} # For high-availability change set to 2-4
+  revisionHistoryLimit: 1
```

Comment:
Could you clarify why the `revisionHistoryLimit` is set to `1` here?

Reply:
Sure. The `revisionHistoryLimit` parameter is used to control how many replica sets the K8s cluster will keep at one time, even if their deployments are removed. This mechanism is designed to have a backup replica set/pod in the cluster in case you want to roll back to an older version. For our use case, I believe we just want to keep the last version of Job Manager when a new version is deployed.